extension/install.rdf

```diff
@@ -6,7 +6,8 @@
     <em:name>Personas Plus</em:name>
     <em:version>1.7.7</em:version>
     <em:description>Your Firefox, your style. Easy-to-change themes.</em:description>
-
+	<em:bootstrap>true</em:bootstrap>
+	<em:unpack>true</em:unpack>
```

Comment:
Sure, just thought maybe you might want to check the add-on in Profile folder after it is compiled. So left it as true. Will remove it.